examples/taint-checking/taint_checking.description

```diff
@@ -1,8 +1,12 @@
-// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. Ballerina programs
-// are resilient to major security vulnerabilities including SQL injection, path manipulation, file manipulation,
-// unauthorized file access, and unvalidated redirect (open redirect).<br/><br/>
+// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. With optional taint checking,
+// Ballerina programs are resilient to major security vulnerabilities including SQL injection,
```

Comment:
Do we need this change? IMO just the part at the end is enough?

Reply:
I think there should be some introduction to what taint checking is, as it's not a widely known thing

Reply:
But isn't that what the next paragraph on "taint analysis" explains? 

I don't think we need the "optional" part here since we're mentioning it in the last paragraph.

Reply:
The thing is without the taint checking is Ballerina programs resilient to major security vulnerabilities. That's why I amended this.

Reply:
Ah, okay. What about something like 
```suggestion
// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. Ballerina programs
// can be resilient to major security vulnerabilities including SQL injection,
```
because we're mentioning that it is disabled by default in the last para?

Was also thinking of
```suggestion
// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. By enabling taint checking,
// Ballerina programs can be made resilient to major security vulnerabilities including SQL injection,
```
but since we haven't spoke about "taint analysis" yet, kinda feels out of place?

Reply:
Updated the description, please check